source/includes/_cvt_events.md

```diff
@@ -7,29 +7,90 @@
 |---|---|
 | GET | /client_reports/cvt_events/[gmaid]?[query_params]|
 
-### API Name: CVT Events
+### API Name: CVT/Web Events
+
+### Summary
+
+This is an API that can be used by customers for whom we are pages of importance to you (CVT/Web Events). All metrics are reported against the CVT/Web Event.
+
+Web Events fall under two categories: 'Request' and 'Submit'. A 'Request' Web Event is usually informational, in some cases, it can have a post-click activity that can generate a lead. A 'Submit' CVT/Web Event is lead-generating. Metrics tied to a 'Request' CVT/Web Event are considered non-qualified, whereas metrics tied to a 'Submit' CVT/Web Event are considered qualified.
+
 ### Usage
-Use GET to retrieve web event activity for all CVT events for a given gmaid.
+Use GET to retrieve web event activity for all CVT/Web Events for a given gmaid.
 
-The data returned will include web events from each campaign with web event name, web event url, and web event interval data of CVT counts, qualified and non-qualified web events, and high web events.
+The data returned will include CVT/Web Events from each campaign with web event name, web event url, and metrics for the interval requested.
 
 ### Parameters
 
 When using the GET method, the results can be filtered using these parameters:
 
 | Param | Required | Function |
 |---|---|---|
-|`start_date`|Yes|Restricts the results to those occurring on or after this date.|
-|`end_date`|Yes|Restricts the results to those occurring on or before this date.|
+|`start_date`|Yes|Restricts the results to metrics occurring on or after this date.|
+|`end_date`|Yes|Restricts the results to metrics occurring on or before this date.|
 |`global_master_campaign_id[]`|No|Restricts results to all campaigns with given master campaign id|
 |`interval_size`|No|Use `calendar_month` or `calendar_week` to roll up the data points into calendar intervals (default is 1 day per interval)|
 |`campaign_status[]`|No|Restrict results to all campaigns with given status values.  Allowed values are `running`, `stopped` or `ended`|
 
-
 To specify a date range:
 
    - Specify start_date and end_date.
 
+### Response Data Details
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`report_type`|String|Report Type.|
+|`report_date`|String|Date Report was Run.|
+|`earliest_date_available`|String|Earliest Date Data is Available.|
+|`start_date`|String|Start Date of Report.|
+|`end_date`|String|End Date of Report.|
+|`time_zone`|String|Time Zone of Report.|
+|`interval_size`|String|Interval Size Report is Broken Into.|
+|`currency`|String|Currency of Report.|
```

Comment:
See earlier note about currency.

Reply:
This has been adjusted to match currency in the Chat Summary, but this report also does not contain monetary values.